org.paraview.ParaView.yaml

```diff
@@ -0,0 +1,191 @@
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app
+      - ./b2 install variant=release link=shared runtime-link=shared cxxflags="$CXXFLAGS" linkflags="$LDFLAGS" -j $FLATPAK_BUILDER_N_JOBS
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.gz
+        sha256: aeb26f80e80945e82ee93e5939baebdca47b9dee80a07d3144be1e1a6a66dd6a
+        
+  - name: hdf5
+    sources:
+      - type: archive
+        url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.12/hdf5-1.12.0/src/hdf5-1.12.0.tar.gz
+        sha256: a62dcb276658cb78e6795dd29bf926ed7a9bc4edf6e77025cd2c689a8f97c17a
+    cleanup:
+      - "/share/hdf5_examples"
+
+  - name: netcdf
+    sources: 
+      - type: archive
+        url: https://github.com/Unidata/netcdf-c/archive/v4.7.4.tar.gz
+        sha256: 99930ad7b3c4c1a8e8831fb061cb02b2170fc8e5ccaeda733bd99c3b9d31666b
+        
+  - name: openmpi
+    sources:
+      - type: archive
+        url: https://download.open-mpi.org/release/open-mpi/v4.1/openmpi-4.1.0.tar.bz2
+        sha256: 73866fb77090819b6a8c85cb8539638d37d6877455825b74e289d647a39fd5b5
+        
+  - name: TBB
+    buildsystem: cmake
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/oneapi-src/oneTBB/archive/2019_U8.tar.gz
+        sha256: 6b540118cbc79f9cbc06a35033c18156c21b84ab7b6cf56d773b168ad2b68566
+      - type: patch
+        path: tbb_cmake.patch
+
+    # Paraview need 32GB of memory to build
+  - name: Paraview
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE:STRING=Release
+      - -DBUILD_TESTING:BOOL=OFF
+      - -DPARAVIEW_USE_QT:BOOL=ON
+      - -DPARAVIEW_USE_PYTHON:BOOL=ON
+      - -DPARAVIEW_ENABLE_WEB:BOOL=ON
+      - -DPARAVIEW_ENABLE_EMBEDDED_DOCUMENTATION:BOOL=ON
+      - -DVTK_PYTHON_OPTIONAL_LINK:BOOL=OFF
+      - -DVTK_OPENGL_HAS_OSMESA:BOOL=OFF
+      - -DPARAVIEW_USE_QTHELP:BOOL=OFF
+      - -DBUILD_EXAMPLES:BOOL=ON
+      - -DPARAVIEW_PLUGINS_DEFAULT:BOOL=ON
+      - -DVTK_SMP_IMPLEMENTATION_TYPE:STRING=TBB
+      - -DVTK_PYTHON_VERSION:STRING=3
+      - -DVTK_PYTHON_FULL_THREADSAFE:BOOL=ON
+      - -DVTK_NO_PYTHON_THREADS:BOOL=OFF
+      - -DQtTesting_INSTALL_NO_DEVELOPMENT:BOOL=ON
+      - -DVTK_BUILD_QT_DESIGNER_PLUGIN:BOOL=OFF
+      - -DPARAVIEW_INSTALL_DEVELOPMENT_FILES:BOOL=ON
+      - -DPARAVIEW_BUILD_WITH_EXTERNAL:BOOL=OFF
+      - -DPARAVIEW_ENABLE_FFMPEG:BOOL=ON
+      - -DPARAVIEW_ENABLE_VISITBRIDGE:BOOL=ON
+      #- -DPARAVIEW_USE_CUDA:BOOL=ON Not sure how to do this
+      - -DPARAVIEW_USE_MPI:BOOL=ON
+    sources:
+      - type: archive
+        url: https://www.paraview.org/files/v5.9/ParaView-v5.9.0.tar.gz
+        sha256: f96c9c996903b295dcfb7db689411d1d0e6a3ccaeb6935c70deb8cc9e5e657c6
+      - type: patch
+        path: fix_appdata_file.patch
+        # ParaView is missing 128x128 icon.
+      - type: patch
+        path: add_icon_128x128.patch
```

Comment:
Ideally you should try to send these patches upstream.

Reply:
Yes, I already did.